sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_extensions.cs

```diff
@@ -293,7 +293,9 @@ public virtual Response SendToGroup(string group, string content, ContentType co
         /// <summary> Check if there are any client connections inside the given group. </summary>
         /// <param name="group"> Target group name, which length should be greater than 0 and less than 1025. </param>
         /// <param name="context">Options specifying the cancellation token, controlling error reporting, etc.</param>
+#pragma warning disable AZC0018 // Protocol method should take a RequestContext parameter called `context` and not use a model as parameter or return types.
         public virtual async Task<Response<bool>> GroupExistsAsync(string group, RequestContext context = default)
+#pragma warning restore AZC0018 // Protocol method should take a RequestContext parameter called `context` and not use a model as parameter or return types.
         {
```

Comment:
I don't know why there is a `RequestContext` and a `Response<bool>`. But this one is GAed. I have to suppress.

Reply:
I think we allow this one for HEAD requests -- it would be worth checking the generator code to confirm my memory that this is what the generator will write in all cases for a HEAD request .  It might make sense to update the PR for this so it doesn't confuse client authors in the future, but I'm happy for us to track this as future work in a separate issue.

Reply:
Yes, you are correct. I fixed in this PR.